module/tools/SystemConfiguration/data/system/default/etc/wpa_supplicant/wpa_supplicant-wlp58s0.conf

```diff
@@ -4,6 +4,12 @@ update_config=1
 fast_reauth=1
 ap_scan=1
 
+network={
+	ssid="NETGEAR89"
+	#psk="fancyriver224"
+	psk=6d1c66448797d8bcdb0ecca60f820d2d4be0e012eea7c3542388cfe0befcfa54
+    priority=3
+}
```

Comment:
:monocle_face:

Reply:
Won't this make the robots connect to the netgear router instead of epsilon-z normally?

Reply:
Normally the router should be off, so it will fallback to the next priority :shrug: